webapp/components/deactivate_member_modal.jsx

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {updateActive} from 'actions/user_actions.jsx';
+import {intlShape, injectIntl, FormattedMessage} from 'react-intl';
+
+import {Modal} from 'react-bootstrap';
+
+import React from 'react';
+
+class DeactivateMemberModal extends React.Component {
```

Comment:
Is it possible to use the `ConfirmModal` component instead of creating a new one? We use that in other places for confirmation